hypex/ab_test/aa_tester.py

```diff
@@ -906,7 +949,28 @@ def experiment_result_transform(self, experiment: pd.Series):
             for i in experiment.index:
                 if i.startswith(f"{tf} "):
                     targets_dict[tf][i[len(tf) + 1 :]] = experiment[i]
-        return pd.DataFrame(targets_dict).T, experiment.iloc[-9:]
+
+        return {
+            "best_experiment_stat": pd.DataFrame(targets_dict).T,
+            "best_split_stat": experiment.iloc[-9:],
+        }
+
+    @staticmethod
```

Comment:
line 984 
now it must be ssp = self.split_splited_data(splited_data)

Reply:
It's strange that the test passed. I'll fix it.